share/sysbuild/cmake/modules/sysbuild_kconfig.cmake

```
@@ -48,6 +48,7 @@ file(TOUCH ${CMAKE_CURRENT_BINARY_DIR}/empty.conf)
 set(APPLICATION_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(AUTOCONF_H             ${CMAKE_CURRENT_BINARY_DIR}/autoconf.h)
 set(CONF_FILE              ${SB_CONF_FILE})
+set(OVERLAY_CONFIG         "${CMAKE_CURRENT_BINARY_DIR}/empty.conf")
```

we should probably support a `SB_OVERLAY_CONFIG` for sysbuild.

```suggestion
set(OVERLAY_CONFIG         "${SB_OVERLAY_CONFIG}")
```

Note, quoting is used to ensure that `OVERLAY_CONFIG` as minimum becomes an empty string, cause otherwise the local scope `OVERLAY_CONFIG` would become undefined and thus the cache variable will be used instead (which we don't want in this case).

---

Sounds good. I wasn't sure if `SB_OVERLAY_CONFIG` would be needed, but I can add it here.